include/galsim/SBInclinedExponentialImpl.h

```diff
@@ -0,0 +1,218 @@
+/* -*- c++ -*-
+ * Copyright (c) 2012-2016 by the GalSim developers team on GitHub
+ * https://github.com/GalSim-developers
+ *
+ * This file is part of GalSim: The modular galaxy image simulation toolkit.
+ * https://github.com/GalSim-developers/GalSim
+ *
+ * GalSim is free software: redistribution and use in source and binary forms,
+ * with or without modification, are permitted provided that the following
+ * conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions, and the disclaimer given in the accompanying LICENSE
+ *    file.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions, and the disclaimer given in the documentation
+ *    and/or other materials provided with the distribution.
+ */
+
+#ifndef GalSim_SBInclinedExponentialImpl_H
+#define GalSim_SBInclinedExponentialImpl_H
+
+#include "SBProfileImpl.h"
+#include "SBInclinedExponential.h"
+#include "LRUCache.h"
+#include "Table.h"
+
+namespace galsim {
+
+    /// @brief A private class that caches the needed parameters for each inclined exponential angle 'i'
+    class InclinedExponentialInfo
+    {
```

Comment:
I don't think there is any use for this class.  As far as I can tell, you don't have any complex calculations to perform that will be true for all `InclinedExponential` objects.  Mostly they will have different values of `h_tani_over_r`, so there would need to be a separate Info object for each Impl object.  Better to just combine these into a single class, `SBInclinedExponentialImpl`.